pep_sphinx_extensions/pep_zero_generator/pep_index_generator.py

```diff
@@ -30,9 +31,29 @@
     from sphinx.environment import BuildEnvironment
 
 
-def create_pep_zero(_: Sphinx, env: BuildEnvironment, docnames: list[str]) -> None:
-    # Sphinx app object is unneeded by this function
-
+def create_pep_json(peps: list[parser.PEP]) -> str:
+    pep_dict = {
+        pep.number: {
+            "title": pep.title,
+            "authors": ", ".join(pep.authors.nick for pep.authors in pep.authors),
```

Comment:
Would it be better to make this an array of nicks, so users don't need to split the string again?

Reply:
Currently, all of the fields are strings (or null), and match the literal value of the headers (minus cleaned-up whitespace). if we're going to do this, it would be inconsistent if we didn't process the other headers as well into more structured formats, which would be a good idea but better left to a future PR. With #2484 , tools can rely on each of the headers to match a certain format, so they are easier and more consistent to work with as strings and can be split with just `.split(",")`, without having to worry much about edge cases.

As mentioned above,

> In the future, if there's any interest, we could actually parse the fields into more structured formats, especially once my PRs land that fully validate their format for much easier and trouble-free internal and external use.